### DISCUSSION TITLE
I'm a Chinese. Why can't dongfanghong-1 satellite be displayed? I searched it for a long time and couldn't find it.

### DISCUSSION THREAD

**Discussion by 1234f503** [Stellarium/stellarium#2524] [Q&A] [answered]
I'm a Chinese. Why can't dongfanghong-1 satellite be displayed? I searched it for a long time and couldn't find it.

**Comment by gzotti** [comment, marked as answer]
Same as #2518.

**Reply by 1234f503** [reply]
However, there is no Dongfanghong 1 in the tle table,

**Reply by alex-w** [reply]
By default we use celestrak as source of TLE and this site published "modern" TLE. If you want to see satellites from 1970 then you need add TLE source for it.
